README.md

```diff
@@ -1,36 +1,282 @@
+
+At a minimum, you will likely include the main entry point for your app here.
+
+This file is processed with Sprockets, so anything included here will be minified.
+
+Do not add library or remote scripts here. Those should be added with `config/high_five.rb` (see below for how to do this).
+
+* `config/high_five/app-platform.js`
+
+This file is a template and *must* be copied to create `app-<platform>.js` for each platform you have which by default are 'ios' and 'android'.
+
+After creating `app-<platform>.js` for each of your platforms, you can delete this file.
+
+TODO: Just create the app-ios.js and app-android.js files instead of app-platform.js. This would get us a little closer to "just working".
+
+* `config/high_five/index.html.erb`
+
+The entry page for your app.
+
+TODO: Why reverse on @stylesheets? Is there a better way to do it?
+
+TODO: Why not reverse on @javascripts? Should it be like @stylesheets or the other way around? Or are they correct the way there are now?
+
+## Configuration
+
+* `config.app_name`
+
+TODO What does this do? Probably sets some values in some places
+
+* `config.app_id`
+
+TODO What does this do? Probably sets some values in some places
+
+* `config.root`
+
+Defaults to `File.join(File.dirname(__FILE__), '..')` which should be the top-level directory of your app.
+
+* `config.destination`
+
+This is to where high five deploys your app which is `www` by default. (TODO Is that true? It's commented out by default so I don't know what it does if you keep it commented out. Use www? Error out?)
+
+If you are using high five with a cordova app, this will need to be set for iOS and Android separately (uncomment the appropriate lines in the platforms section of the config).
+
+* `config.windows!`
+
+Use this if you're on windows and having problems with a strange ExecJS RuntimeError.
+
+* `config.compass_dir`
+
+If set, high five will run `compass compile` in the given directory before doing anything else.
+
+This makes it possible, for example, to write sass in `resources/sass/app.sass` and have it compiled to `resources/css/app.css` which can then be included into your app with the `stylesheets` method (see below).
+
+NB: Make sure you have specified your compass config in `config.rb` in this directory! (TODO Is this necessary or is there some sort of default config that compass will use? Does it do anything useful? Does it error out?)
+
+TODO Do we include this file by default? If we do, does it have good defaults? If we do, make sure to write some words stating as much and link to a good overview of the compass config.
+
+* `config.minify`
+
+TODO What does this do? The default config just has "# production.minify :uglifier # or :yui". Does it work/actually do something/error out if I uncomment the setting and use it?
+
+* `config.manifest`
+
+Generate app cache manifest. TODO What does that mean? When should I use this option?
```

Comment:
http://www.w3schools.com/html/html5_app_cache.asp you use it in some cases if you're deploying to the web